Fitz/Features/Accounts/Commands/AccountSlashCommands.cs

```diff
@@ -51,8 +50,6 @@ public AccountSlashCommands(BotContext db, AccountService accountService, BankSe
             #endregion Account Creation Interactions
         }
 
-        #region Signup
-
         [SlashCommand("signup", "Just sign this form.")]
```

Comment:
This is the original command. Everything that it did is how it works in main currently. You'll probably want to have another screen up with this exact file to see all of the business/domain logic that it has. 

I feel that you'll follow the updated one easier.
AccountsRework/Create/Discord/SignupAdapter.cs



Questions:
L60 - I'm giving the user an account creation bonus. If this is business logic, should it exist in the CreateAccountConductor? AKA Facade.

L71 - I'm giving the user a new role. I'm thinking this becomes a job that is fired by the CreateAccountConductor once we know an account has been created. Where do you think it should go?

Reply:
Ignore everything else in this file such as the other commands. I only focused on refactoring the first command anyone would ever run which is /signup